docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md

```diff
@@ -21,6 +21,26 @@ This guide describes the best practices and tuning approaches to scale Rancher s
 
 When scaling up Rancher, one typical bottleneck is resource growth in the upstream (local) Kubernetes cluster. The upstream cluster contains information for all downstream clusters. Many operations that apply to downstream clusters create new objects in the upstream cluster and require computation from handlers running in the upstream cluster.
 
+### Minimizing third-party software that runs on the Upstream Cluster
+
+Running Rancher at scale can put significant load on internal Kubernetes components such as `etcd` or `kubeapiserver`, therefore issues may arise if third party software interferes with the performance of those components or Rancher itself.
+
+In general, every third party piece of software carries a risk of interference, therefore to avoid performance issues it is recommended to limit software running on the Upstream Cluster to Kubernetes system components and Rancher itself as close as possible.
+
+Within that premise, software in the following categories is generally safe to use:
+ * CNIs
+ * Cloud controller managers
+ * Observability and monitoring tools (see an exception below)
+ * Cluster API components
+ * Rancher internal components such as Fleet
+ * Rancher extensions
```

Comment:
```suggestion
 * Rancher internal components, such as Fleet
 * Rancher extensions
 * Cluster API components
 * CNIs
 * Cloud controller managers
 * Observability and monitoring tools (with the exception of prometheus-rancher-exporter)
```

Nitpicky listing Rancher/cluster components up top to reassure users that our own components won't mess with performance